tests/dragonfly/acl_family_test.py

```diff
@@ -684,18 +684,21 @@ async def publish_worker(client):
         logging.debug("Starting publish_worker")
         for i in range(0, 10):
             logging.debug(f"publisher iteration: {i}")
-            await client.publish("channel", "message")
+            await client.publish("channel", f"message{i}")
 
     async def subscribe_worker(channel: aioredis.client.PubSub):
         logging.debug("Starting subscribe_worker")
-        total_msgs = 1
+        total_msgs = 0
         async with async_timeout.timeout(10):
             while total_msgs != 10:
                 try:
                     res = await channel.get_message(ignore_subscribe_messages=True, timeout=5)
+                    if res is None:
```

Comment:
so `get_message` can return `None` even if it doesn't timeout :scream:

Reply:
yep, I've seen it, but it still doesn't explain everything.
See the failure, it shows receiving `message4` while expecting `message0`:

```
2024-09-23T10:58:07.4929895Z >                   assert res["data"] == f"message{total_msgs}"
2024-09-23T10:58:07.4930555Z E                   AssertionError: assert equals failed
2024-09-23T10:58:07.4931314Z E                     'message�^4'  'message�^0'
```

(ignore unprintable coloring characters)

Reply:
wow